api/routes.py

```diff
@@ -147,12 +147,19 @@ def submit_manifest_route(schema_url):
 
     data_type = connexion.request.args["data_type"]
 
+    input_token = connexion.request.args["input_token"]
+
+    if input_token == 'None':
+        input_token = None
```

Comment:
Try with input_token nullable set to true but w/o this section. If it doesn't work we can note that. 

In any case, after this test  we want to set [nullable to false](https://github.com/Sage-Bionetworks/schematic/pull/597/files#r823104216) in this case